core/build.gradle

```diff
@@ -20,6 +20,10 @@ android {
             }
         }
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+    }
```

Comment:
Hmm, the documentation says "Take care when setting the returnDefaultValues property to true. The null/zero return values can introduce regressions to your tests, which are hard to debug and might allow failing tests to pass. Only use it as a last resort". Do we really want to enable this?